staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2163,6 +2179,24 @@ type RemoveVolumeOptions struct {
 	DryRun []string `json:"dryRun,omitempty"`
 }
 
+// AddInterfaceOptions is provided when dynamically hot plugging a network interface
+type AddInterfaceOptions struct {
+	// NetworkName indicates the name of the network to which the interface will be connected
+	NetworkName string `json:"networkName"`
+
+	// InterfaceName indicates the name of the interface being plugged into the guest
+	InterfaceName string `json:"interfaceName"`
+}
+
+// RemoveInterfaceOptions is provided when dynamically hot unplugging a network interface
+type RemoveInterfaceOptions struct {
```

Comment:
Isn't this PR focused on hotplug only?

Reply:
Yes. The server rejects these requests since the feature isn't provided. 

If you prefer I'll remove this altogether, and introduce these when the unplug feature is provided.

Reply:
Removed all the unplug related code.